addons/io/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/AbstractServlet.java

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.io.neeo;
```

Comment:
Imho most of your classes should be moved in an "internal" package and not be exposed.